deploying-ens-on-a-private-chain.md

```diff
@@ -75,7 +75,7 @@ Above, we first create a new top-level domain, "resolver", then set its resolver
 
 ## Deploy a Registrar
 
-So far, domains can only be registered manually by the owner of the registry's root node. Fortunately, contracts can also own nodes. This means we can set up a registrar contract as the owner of a node, e.g. "test", in the registry which enables it to distribute subdomains such as "mycontract.test". It allows us to have custom, on-chain logic which governs domain allocation. Once we own a \(sub-\)node we are free to repeat this process and set up another registrar. If you are part of the "myorg" organisation you could register "myorg.test" and let it point to your custom registrar which only allows certified members of your organisation to claim subdomains such as "bob.myorg.test". For our private network, we'll use the simple 'first come, first served' [FIFSRegistrar](https://github.com/ensdomains/ens/blob/master/contracts/FIFSRegistrar.sol), and set it as the owner of the top-level domain "test" in our migration script:
+So far, domains can only be registered manually by the owner of the registry's root node. Fortunately, contracts can also own nodes. This means we can set up a registrar contract as the owner of a node, e.g. "test", in the registry which enables it to distribute subdomains such as "mycontract.test". It allows us to have custom, on-chain logic which governs domain allocation. Once we own a \(sub-\)node we are free to repeat this process and set up another registrar. If you are part of the "myorg" organisation you could register "myorg.test" and let it point to your custom registrar which only allows certified members of your organisation to claim subdomains such as "bob.myorg.test". For our private network, we'll use the simple 'first come, first served' [FIFSRegistrar](https://github.com/ensdomains/ens-contracts/blob/master/contracts/registry/FIFSRegistrar.sol), and set it as the owner of the top-level domain "test" in our migration script:
 
```

Comment:
This whole section may be out of date. You may simply point to https://github.com/ensdomains/ensjs-v3/tree/main/packages/ens-test-env that comes with [deployment scripts](https://github.com/ensdomains/ensjs-v3/tree/main/packages/ensjs/deploy) or @jefflau should review the code given he recently did testnet deployment

Reply:
I have added an internal doc to Leon's new doc. We could add it here, but it would likely need some explanation and expanding, but works as an internal doc for now

Reply:
I'm just going to keep it as-is for now - it's not any worse than it was before and it sounds like we'll have a major upgrade soon, so doesn't seem like a blocker to the other changes in this PR.